webapp/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactComponent as IconArrowBack } from './icons/arrow-back.svg';
```

Comment:
As before, I'd move this into a `common` folder since this components will be shared by multiple screens.

`webapp/src/common/components/Icon...`